autotest/ogr/ogr_dxf.py

```diff
@@ -3628,6 +3628,24 @@ def test_ogr_dxf_54():
             pytest.fail('Wrong visibility on feature %d' % number)
 
 
+###############################################################################
+# Test hidden objects in blocks
+
+
+def test_ogr_dxf_55():
+
+    gdal.SetConfigOption('DXF_MERGE_BLOCK_GEOMETRIES', 'FALSE')
```

Comment:
``with gdaltest.config_option('DXF_MERGE_BLOCK_GEOMETRIES', 'FALSE'):`` will take care of restoring the config option to its initial value